src/utils/fileName/fileName.tsx

```diff
@@ -1,11 +1,19 @@
-import { Config } from "../../types";
+interface generateFileNameI {
+  network?: string;
+  fileName: string;
+  extension: string;
+  timestamp?: string;
+}
 
-export const generateFileName = (
-  config: Config | undefined,
-  fileName: string,
-  extension: string
-): string => {
-  return `${fileName}${config?.network === "homestead" ? "" : `-${config?.network}`}.${extension}`;
+export const generateFileName = ({
+  network,
+  fileName,
+  extension,
+  timestamp,
```

Comment:
Since u added the timestamp into the generateFileName function, i think we can remove generateErrorLogFileName already, cos it is only adding timestamp to the file name.

Maybe can refactor this part then should be ok liao

Reply:
ok updated